manifests/gcp_marketplace/schema.yaml

```diff
@@ -175,24 +175,20 @@ properties:
     default: false
     description: |-
       Kubeflow Pipelines default to use docker executor. Select this option to
-      use [the emissary executor (Alpha)](https://github.com/kubeflow/pipelines/issues/6249).
+      use the emissary executor.
 
-      Refer to migrating to emissary executor documentation for migration instructions.
+      Warning, docker executor does not work on Google Kubernetes Engine (GKE) 1.19+
```

Comment:
"Docker executor does not work on Google Kubernetes Engine (GKE) 1.19+ by default, as docker container runtime is no longer the default container runtime; docker container runtime will be deprecated on GKE 1.20+."